### ISSUE TITLE
Pytest-plugins error - conftest not on rootdir

### ISSUE THREAD

**Issue by dimon222** [pytest-dev/execnet#91]
```
Defining 'pytest_plugins' in a non-top-level conftest is no longer supported because it affects the entire directory tree in a non-explicit way.
  /tmp/py-execnet/src/execnet-1.5.0/doc/example/conftest.py
Please move it to a top level conftest file at the rootdir:
  /tmp/py-execnet/src/execnet-1.5.0
For more information, visit:
  https://docs.pytest.org/en/latest/deprecations.html#pytest-plugins-in-non-top-level-conftest-files
```

**Comment by nicoddemus** [comment]
Hi, can you provide more details please? Which command line are you executing?

**Comment by dimon222** [comment]
python -m pytest

**Comment by nicoddemus** [comment]
I believe if you execute `pythoh -m pytest testing` will work as well, can you please confirm?

**Comment by dimon222** [comment]
works with `python -m pytest testing`

**Comment by nicoddemus** [comment]
We can make `python -m pytest` work by adding `testpaths = testing` to the `pytest` section in `tox.ini`.

Would you like to contribute a PR? 😉

**Comment by nicoddemus** [comment]
Closed by #92, thanks @dimon222!
